hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -382,6 +382,13 @@
   /** Default maximum file size */
   public static final long DEFAULT_MAX_FILE_SIZE = 10 * 1024 * 1024 * 1024L;
 
+  /** Conf key for if we should sum overall region files size when check to split */
+  public static final String OVERALL_HREGION_FILES =
```

Comment:
Should we use 'hregion' or 'region'? I'm not sure, just ask.

Reply:
I used region first, and changed to hregion when i saw the surround usage such as "HREGION_COMPACTIONDIR_NAME".